common/autoconf/platform.m4

```diff
@@ -67,7 +70,7 @@ AC_DEFUN([PLATFORM_EXTRACT_VARS_FROM_CPU],
       VAR_CPU_ENDIAN=big
       ;;
     powerpc64le)
-      VAR_CPU=ppc64
+      VAR_CPU=ppc64le
```

Comment:
Any idea why OpenJDK builds use a ppc64le directory already without this fix?

Reply:
No idea.

Reply:
I have concerns about merging this. As OpenJDK uses a ppc64le directory without this change, it implies there is something wrong with the OpenJ9 build process that can be fixed without patching the OpenJDK code.

Reply:
The new version I pushed earlier today changes only 'closed' (and `generated-configure.sh`) files.